### PR TITLE
Backport changes in system requirements from 10.3 to 10.4

### PR DESCRIPTION
**Pull request by settermjd**
This PR syncs the changes in 10.3 with 10.4.

### CODE REVIEW

File: modules/admin_manual/pages/installation/system_requirements.adoc [hunk 1]
```diff
@@ -120,21 +94,34 @@ xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-read
 * Xen
 * KVM
 
-=== Desktop
+=== Mobile OS
```
Comment (phil-davis) [inline modules/admin_manual/pages/installation/system_requirements.adoc]: If 10.3 branch says "Mobile OS" here, then IMO the heading is wrong in 10.3, and needs to be fixed there.

File: modules/admin_manual/pages/installation/system_requirements.adoc [hunk 2]
```diff
@@ -97,21 +79,13 @@ with a MySQL or MariaDB database:
 xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-storage-engine[MySQL / MariaDB storage engine])
 * `READ COMMITED` transaction isolation level (See: 
 xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-read-commited-transaction-isolation-level[MySQL / MariaDB `READ COMMITED` transaction isolation level])
-====
-
 
-=== Mobile
```
Comment (phil-davis) [inline modules/admin_manual/pages/installation/system_requirements.adoc]: Did we want to lose the "Mobile" section?

Comment (settermjd) [reply modules/admin_manual/pages/installation/system_requirements.adoc]: Correct. I'll add it back in.

File: modules/admin_manual/pages/installation/system_requirements.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/pages/installation/system_requirements.adoc]: I just corrected it in this branch and will correct it in 10.3

**Pull request merged**